plugins/polio/api.py

```diff
@@ -2175,6 +2175,10 @@ def create(self, validated_data):
         user = self.context["request"].user
         validated_data["account"] = user.iaso_profile.account
 
+        expiration_date = validated_data.get("expiration_date")
```

Comment:
This will work, but it seems to me that we should use the serializers to do this kind of validation, like here: https://stackoverflow.com/questions/30953481/django-rest-framework-how-to-make-model-serializer-fields-required 

(not a blocking remark)

Reply:
Actually, I'm wondering if changing at the database level does not already imply that the ModelSerializer will send back that the field is required if provided empty. That's what would make sense to me

Reply:
@madewulf I was looking into it and it seems something like on the serializer would work

```pythin
extra_kwargs = {"expiration_date": {"required": True, "allow_null": False, "allow_blank": False}}
```

Not sure because it seems to depend on the version of DRF

Reply:
@madewulf yes I was thinking that too, I guess it needs to be tested ? Not sure how it goes while defining the field as a "read_only" TimestampField on the serializer as it at the moment ....

Reply:
Indeed, the check from DB is enough. I removed it